docs/CHANGELOG.md

```diff
@@ -12,6 +12,13 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 <!-- PUT THE NEW CHANGELOG ENTRY RIGHT BELOW THIS -->
 <!-------------------------------------------------->
 
+## Upcoming
+
+* The raw count matrix is again rounded to integer values, in agreement with the documentation
```

Comment:
If I'm right that this only impacted sc/snRNA-seq and if using code styling for those words is consistent with what we do elsewhere (I'm pretty sure it is):

```suggestion
* Recent versions of the raw count matrices in `SingleCellExperiment` and `AnnData` objects were not rounded. 
The raw counts are now rounded to integer values in agreement with the documentation.
```